CScript.cs

```diff
@@ -2505,15 +2547,31 @@ public bool UpdateFromBinary(byte[] binData)
             numInstructions = System.BitConverter.ToUInt32(binData.AsSpan<byte>(offset, 4));
             numInstructionsOffset = (uint)offset;
             offset += 4;
-
             InstructionBlockOffset = offset;
 
+            if (numInstructions < 0 | numInstructions > binData.Length)
```

Comment:
Bitwise "or" works, technically, but should be replaced with a logical OR = ||

Reply:
Resolved - the first half of these checks has been removed as they now all pertain to unsigned integers, so OR is no longer used at all.